code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm

```diff
@@ -116,14 +116,14 @@
 	for(var/atom/movable/ravaged AS in atoms_to_ravage)
 		if(!(ravaged.resistance_flags & XENO_DAMAGEABLE))
 			continue
-		if(!ravaged.anchored)
-			step_away(ravaged, X, 1, 2)
-		if(!ishuman(ravaged))
+		if(!ishuman(ravaged) && !ravaged.anchored)
```

Comment:
You've removed the is human check before typecasting ravaged to human below. This means for example, you'll runtime when you ravage a wall because it's not human but is anchored, so will be typecast as a human.

Reply:
Walls are not `/atom/movable`, so this shouldn't come up. It also slashes non human atoms just fine.

Reply:
Whoops bad example, but it should still apply to anchored atoms, unless I'm illiterate which is possible.